### PR TITLE
docs: add guide to integrate with WalletKit as a signer

### PR DESCRIPTION
**Pull request by tusharsoni**
# Pull Request Checklist

[N/A] Did you add new tests and confirm existing tests pass? (`yarn test`)
- [x] Did you update relevant docs? (docs are found in the `site` folder, and guidelines for updating/adding docs can be found in the [contribution guide](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md))
- [x] Do your commits follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [x] Does your PR title also follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
[N/A] If you have a breaking change, is it [correctly reflected in your commit message](https://www.conventionalcommits.org/en/v1.0.0/#examples)? (e.g. `feat!: breaking change`)
- [x] Did you run lint (`yarn lint:check`) and fix any issues? (`yarn lint:write`)
- [x] Did you follow the [contribution guidelines](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md)?

---

### Detailed summary
- Added `WalletKit` to the sidebar menu in the documentation.
- Added `WalletKit` as an example in the "Some MPC signers provide recovery services" section.
- Added `WalletKit` as an example in the "MPC (Multi-Party Computation)" section.
- Added a new integration guide for using `WalletKit` as a signer with Account Kit.
- Added an image for `WalletKit` in the integration guide.
- Added code snippets and instructions for installing and setting up `WalletKit` in the integration guide.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by tusharsoni** [comment]
Hi team, this PR adds [WalletKit](https://walletkit.com/) to the signer guides alongside similar services. We're here to answer any questions. Thanks!

**Review by moldy530 — COMMENTED** [review]
Hey! thanks for your contribution, sorry for the delay on review here!

This overall LGTM, but have you seen https://accountkit.alchemy.com/smart-accounts/signers/contributing.html

it's up to you if you want to just keep it as a guide or if you want to add support within the SDK directly to make it easier for devs to get up and running. Lemme know and I'm happy to get this merged in either way

**Comment by tusharsoni** [comment]
Thanks for the review, @moldy530!

I looked into adding support within the SDK directly. My understanding is that since WalletKit's library exports an EIP-1193 compliant provider, it is simpler to use `WalletClientSigner`. Lmk if I missed anything, definitely want to make it easier for devs here.

If this looks good to you, let's get this merged and can continue discussion on how to add support within the SDK directly if needed. Thanks again!

**Comment by moldy530** [comment]
Yeah makes sense to me, I'll get this merged in tomorrow and deployed to the V2 docs site. We are working on V3 which changes the signature for provider creation but I'll update this doc when we get closer to releasing those docs

**Comment by moldy530** [comment]
hey @tusharsoni I've not had a chance to get this merged in before we merged the v3 changes into main. do you mind rebasing on main and integrating these changes with the v3 changes? the signer guide and apis for creating a client should be stable now. I've updated all of the old signer docs to match the new apis so feel free to use any of those as a base. sorry for the thrash here!

**Comment by tusharsoni** [comment]
No problem, @moldy530. Updated the PR and rebased on main. Let me know if everything looks good!

File: site/signers/guides/walletkit.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  (rpcClient) =>
+    new LightSmartContractAccount({
+      // Set the EOA wallet, created using WalletKit, the owner for 
+      // this smart contract account
+      owner: signer,
+      chain: rpcClient.chain,
+      factoryAddress: getDefaultLightAccountFactoryAddress(rpcClient.chain),
+      rpcClient,
+    })
+)
```
Comment (moldy530) [inline site/signers/guides/walletkit.md:125]:
```suggestion
import { createModularAccountAlchemyClient } from "@alchemy/aa-alchemy";
import { sepolia } from "@alchemy/aa-core";

const client= await createModularAccountAlchemyClient({
  apiKey: "<Alchemy-API-Key>",
  chain: sepolia,
   owner: signer,
});
```

**Review by moldy530 — COMMENTED** [review]
Only one small change, otherwise LGTM

File: site/signers/guides/walletkit.md (hunk 1 above)
Comment (tusharsoni) [reply site/signers/guides/walletkit.md]: Updated!

**Review by moldy530 — DISMISSED** [review]

**Comment by tusharsoni** [comment]
Looks like there was a lint error - just fixed it

**Review by moldy530 — DISMISSED** [review]

**Comment by tusharsoni** [comment]
[x] Resolved merge conflict
--
There is a build error with connecting to sepolia. Seems to be unrelated to my changes. Lmk if there's anything I can do to help fix / debug it.

**Review by moldy530 — APPROVED** [review]
Yeah the unresolved url thing is on our end LGTM!

**Pull request merged**